e2e_test/background_ddl/sim/basic.slt

```diff
@@ -26,23 +29,20 @@ CREATE MATERIALIZED VIEW m3 as SELECT * FROM t;
 statement error
 CREATE MATERIALIZED VIEW m3 as SELECT * FROM t;
 
-# Wait for background ddl to finish
-sleep 30s
-
 query I
 select count(*) from m1;
 ----
-10000000
+200000
 
 query I
 select count(*) from m2;
 ----
-10000000
+200000
 
 query I
 select count(*) from m3;
 ----
-10000000
+200000
 
```

Comment:
Why are these results `10000000` before?

Reply:
This test wasn't running previously (intentionally, since there were some issues.)

Now that it's running we fix the numbers.